src/fabric/include/fabric2.hrl

```diff
@@ -57,7 +57,10 @@
 -define(PDICT_TX_ID_KEY, '$fabric_tx_id').
 -define(PDICT_TX_RES_KEY, '$fabric_tx_result').
 -define(PDICT_ON_COMMIT_FUN, '$fabric_on_commit_fun').
+-define(PDICT_FOLD_ACC_STATE, '$fabric_fold_acc_state').
+
 -define(COMMIT_UNKNOWN_RESULT, 1021).
+-define(TRANSACTION_TOO_OLD, 1007).
```

Comment:
Super minor nit, but should we order by error code and make a comment as such? Seems like it might be easier to match between other definitions of these in fdb/erlang bindings and such.

Reply:
Good idea